src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -298,6 +298,40 @@ object SampleWdl {
     override lazy val rawInputs = Map(ThreeStep.PatternKey -> "." * 10000)
   }
 
+  object WorkflowOutputsWithFiles extends SampleWdl {
+    override def wdlSource(runtime: String = "") =
+      """
+        task A {
+          command {
+            echo "Enfin un peu de francais pour contrer ce raz-de-marée anglais !" > out
+            echo "Jacques Chirac fait du jetski sur la Seine en costume traditionnel russe" > out2
```

Comment:
Haha indeed !
http://m.memegen.com/hsx975.jpg (translate "poor guy")
I actually forgot about that I didn't intend to PR this test as is :p